src/services/routes/org.routes.spec.ts

```diff
@@ -37,7 +37,7 @@ describe('org routes', () => {
   configureMockEnvVars(REQUIRED_SERVER_ENV_VARS);
   const getTestServer = setUpTestServer();
   let serverInstance: FastifyTypedInstance;
-  beforeEach(() => {
+  beforeAll(() => {
```

Comment:
The original `beforeEach()` was correct. This must be working by chance -- Note that you're now caching the instance across tests.